regolith/schemas.py

```diff
@@ -162,6 +163,38 @@
             "status": "pending",
             "title": "SymPy 1.1 Release Support",
         },
+        {
```

Comment:
Thanks for adding this.  We will have to modify the output in the tester of ``resume`` and ``cv`` to make it pass tests.  It is probably easier if I just show you that.  

Since you are going to the trouble to add a new exemplar grant, it makes sense to me if we add all the new fields I have added over time (e.g., the most recent one I sent you).  Before doing that though, again, we should chat as there is an unmerged branch that may contain some of those.  It is fixing up old problems that are not your fault, but if we don't take the opportunity to fix them now when they come up, we never will and the software won't improve.